lib/Sema/DerivedConformance/DistributedActor.cpp

```diff
@@ -1,8 +1,8 @@
-//===--- DerivedConformanceActor.cpp - Derived Actor Conformance ----------===//
+//===--- DerivedConformance/Actor.cpp ---------------------------*- C++ -*-===//
```

Comment:
I'm not a fan of having multiple files called Actor.cpp in the codebase. For example, if you have both open in different tabs, there's not a quick way to differentiate which is which.

Reply:
Let’s keep the old names then.